101-event-grid-cloudevents/azuredeploy.json

```diff
@@ -0,0 +1,189 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "location": {
+      "type": "string",
+      "metadata": {
+        "description": "The region where resources are deployed"
+      },
+      "defaultValue": "[resourceGroup().location]"
+    },
+    "logicAppName": {
```

Comment:
add a defaultValue - use uniqueString as needed

Reply:
`location` is not a unique string, it's region and mostly follows the resource group location. Do you still want me to change it to a specific region (australiasoutheast, for example) or, alternatively, provide a list of regions AND `resourceGroup().location`? There are many templates using `resourceGroup().location` as a default value.

Reply:
Done